docs/source/kedro_project_setup/configuration.md

```diff
@@ -29,6 +29,27 @@ This recursively scans for configuration files firstly in the `conf/base/` (`bas
   * file is located in a sub-directory whose name is prefixed with `catalog`
 * *And* file extension is one of the following: `yaml`, `yml`, `json`, `ini`, `pickle`, `xml` or `properties`
 
+This logic is specified by `config_patterns` in the [ConfigLoader](/kedro.config.ConfigLoader) and [TemplatedConfigLoader](/kedro.config.TemplatedConfigLoader) classes. By default those patterns are set as follows for the configuration of catalog, parameters, logging and credentials:
+```python
+config_patterns = {
+    "catalog": ["catalog*", "catalog*/**", "**/catalog*"],
+    "parameters": ["parameters*", "parameters*/**", "**/parameters*"],
+    "credentials": ["credentials*", "credentials*/**", "**/credentials*"],
+    "logging": ["logging*", "logging*/**", "**/logging*"],
+}
+```
+
+The configuration patterns can be changed by setting the `CONFIG_LOADER_ARGS` variable in [`src/<package_name>/settings.py`](settings.md). You can change the default patterns as well as add additional ones, for example, for Spark configuration files.
+This example shows how to load `parameters` if your files are using a `params` naming convention instead of `parameters` and how to add patterns to load Spark configuration:
```

Comment:
```suggestion
This example shows how to load `parameters` if your files are using a `params` naming convention instead of `parameters` and how to add patterns to load Spark configuration:

```

Reply:
Same here